# Pick up EB CXXFLAGS
# September 11th 2017 by B. Hajgato (Free Univeristy Brussels - VUB)
--- last-869/makefile.orig	2016-12-13 07:22:11.000000000 +0100
+++ last-869/makefile	2017-09-11 14:14:17.989174000 +0200
@@ -1,4 +1,4 @@
-CXXFLAGS = -O3 -std=c++11 -pthread -DHAS_CXX_THREADS
+CXXFLAGS += -pthread -DHAS_CXX_THREADS
 all:
 	@cd src && $(MAKE) CXXFLAGS="$(CXXFLAGS)"
 
